src/libMessage/ZilliqaMessage.proto

```diff
@@ -1060,3 +1060,11 @@ message SeedSetHistoricalDB
     ByteArray signature        = 3;
 
 }
+
+// For removing sender from local blacklist
+message NodeRemoveFromBlacklist
+{
+    ByteArray ipAddress    = 1;
```

Comment:
Need additional information to ensure no replay attack can be done. 

For instance, sig(ipAddress, dsepoch_no)
Ignore request if it exceeds 1 ds epoch.